satellite/console/service.go

```diff
@@ -611,6 +646,12 @@ func (s *Service) DeleteProjectMembers(ctx context.Context, projectID uuid.UUID,
 			continue
 		}
 
+		// abort if one of the members payment info
+		// attached to this project
+		if projPaymentInfo.PayerID == user.ID {
+			return errs.New("member with atached payment can't be deleted")
```

Comment:
Typo: `attached` and `can not` instead of `can't`

Reply:
Also, does that ensure you cannot delete yourself of the Project?

Reply:
it ensures you can't delete member which payment is set as project default. @Barterio is working on adding credit cards so he will add functionality to add/change payment methods to `console`

Reply:
for now it is projects creator default payment method